special-pages/pages/new-tab/types/new-tab.ts

```diff
@@ -55,6 +55,41 @@ export type Expansion = "expanded" | "collapsed";
  * Generic Animation configuration
  */
 export type Animation = None | ViewTransitions | Auto;
+export type Suggestion =
```

Comment:
If you give these a 'title' in the schema, the generator will pull them out into separate types. Purely so that in the generated docs it makes for a nice link when you can point to a type with an identity. (Eg: native folk will likely model smaller data structures for each, since their languages don't have the same type- system capabilities as us :)

Reply:
Good tip, done 👍